pages/checkout.vue

```diff
@@ -149,11 +149,16 @@ export default {
   setup() {
     const nuxt = useNuxtApp()
     const countries = nuxt.nuxt2Context.$config.countries
+    const paymentType = nuxt.nuxt2Context.$config.paymentAction.paymentType
```

Comment:
remove "paymentType" from the config, the component should emit this

Reply:
Done